site/src/pages/RoadmapPage/Planner.tsx

```diff
@@ -61,15 +65,16 @@ const Planner: FC = () => {
     else {
       validatePlanner();
 
+      // TODO: idk if this works properly for multiplanner
```

Comment:
I've found an edge case. Sometimes the unsaved prompt doesn't appear if you change the name of the planner. To reproduce: load up the page for the first time, changing the name of your current planner, attempt to leave or reload. No prompt will block you.

Reply:
Cannot reproduce

https://github.com/icssc/peterportal-client/assets/54484616/ea04ed99-6457-4164-bda4-52a3fbfbde0e

Reply:
https://github.com/icssc/peterportal-client/assets/8922227/2bafe14f-bead-4200-a06a-63beea115055